pkg/cmd/docs-issue-generation/docs_issue_generation_test.go

```diff
@@ -3062,6 +3043,7 @@ Release note (sql change): Import now checks readability...`,
 	}
 
 	for _, tc := range testCases {
+		skip.WithIssue(t, 110682, "Flakes")
```

Comment:
Please put this at the top of the function, before any other logic. It's just a bit odd to have it here in the middle of the loop.